src/views/Definition/Definition.jsx

```diff
@@ -1,13 +1,161 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { useParams } from 'react-router-dom';
+import { useTracking } from 'react-tracking';
 
-import { SearchBox } from '../../components';
+import { DefinitionItem, SearchBox } from '../../components';
+import { useAppPaths, useCustomQuery } from '../../hooks';
+import { getDrugDefinition } from '../../services/api/actions';
+import { useStateValue } from '../../store/store.js';
 
 const Definition = () => {
+	// Pull in the paths we are going to need on this view.
+	const { DefinitionPath } = useAppPaths();
+	const params = useParams();
+	const { idOrName } = params;
+	const queryResponse = useCustomQuery(getDrugDefinition({ idOrName }));
+	const [drugDefinition, setDrugDefinition] = useState();
+	const [drugDefinitionLoaded, setDrugDefinitionLoaded] = useState(false);
+
+	// Get items passed into index.js and stored in the context.
+	const [
+		{
+			analyticsName,
+			altLanguageBasePath,
+			basePath,
+			baseHost,
+			canonicalHost,
+			dictionaryTitle,
+			siteName,
+			language,
+		},
+	] = useStateValue();
+
+	// Get a reference to the tracking function for
+	// analytics.
+	const tracking = useTracking();
+
+	useEffect(() => {
+		if (drugDefinitionLoaded) {
+			tracking.trackEvent({
+				// These properties are required.
+				type: 'PageLoad',
+				event: 'DrugDictionaryApp:Load:Definition',
+				analyticsName,
+				dictionaryTitle,
+				name:
+					canonicalHost.replace('https://', '') +
+					DefinitionPath({
+						idOrName: drugDefinition.payload.prettyUrlName
+							? drugDefinition.payload.prettyUrlName
+							: drugDefinition.payload.termId,
+					}),
+				title: dictionaryTitle,
+				metaTitle: `Definition of ${drugDefinition.payload.name} - ${dictionaryTitle} - ${siteName}`,
+				term: drugDefinition.payload.name,
+				id: drugDefinition.payload.termId,
+				// Any additional properties fall into the "page.additionalDetails" bucket
+				// for the event.
+			});
+		}
+	});
+
+	useEffect(() => {
+		if (queryResponse.payload && !drugDefinitionLoaded) {
+			setDrugDefinition(queryResponse);
+			setDrugDefinitionLoaded(true);
+		}
+	}, [queryResponse, setDrugDefinition]);
+
+	/**
+	 * Helper function to get href lang links IF there is an
+	 * alternate language.
+	 */
+	const getHrefLangs = () => {
```

Comment:
@arcepaul I think this should be killed since this app does not support multiple languages (yet, and no plans for the future.) What say you?

If the code does stay, then it needs to have working integration tests.